other/field-types/joinByArrayReverse.md

```diff
@@ -37,6 +31,9 @@ If desired, we can specify `relationship` and `relationshipsField` just as we wo
 |---|---|---|---|
 | name | `string` | | Sets the name of the field to join with (must begin with `_`) |
 | label | `string` | | Sets the label of the field that the user sees |
-| withType | `string` | | The name of the related type, if it differs from the name of the join |
-| reverseOf | `string` | | Set to the name of the join you are reversing (optional) |
+| withType | `string` | | The name of the related type, if it differs from the name of the join. If you do not set `withType`, then the name of the join must match the name of the related type, with a leading `_` added.  || reverseOf | `string` | | Set to the name of the join you are reversing (optional) |
```

Comment:
Looks like a missing line break here. Markdown seems to understand it, though.

Reply:
Fixed